### ISSUE TITLE
upgrade 3.2.0 broke editor

### ISSUE THREAD

**Issue by wixaw** [overleaf/overleaf#1043]
Hello

When I want to upgrade to 3.2.0, the editor does not display text but compiles PDFs fine
If I don't put the mongo database and I restart the container the editor works
In the meantime I have downgraded to 3.1.1 it works

**Comment by mserranom** [comment]
Hi! 

We've identified this problem when:

- Using Firefox
- Loading a projects with a `user` which has `betaProgram: true` in their DB record.

Would you mind confirming if this is your case?

**Comment by wixaw** [comment]
Ah yes ! i activated beta to help a collaborator ! 
I use Chrome and Brave 
But it's impossible to disable now with 3.1.1 :/

**Comment by mserranom** [comment]
Running the following mongo update should fix the issue (and hopefully will also be fixed in the next release):

```javascript
db.users.updateMany({}, {$set: {betaProgram: false}})
```

If you're using the [toolkit](https://github.com/overleaf/toolkit):
```
toolkit $ bin/mongo
MongoDB shell version v4.0.28
Welcome to the MongoDB shell.
For interactive help, type "help".
> db.users.updateOne({email:"user@example.com"},{$set: {betaProgram:false}})
{ "acknowledged" : true, "matchedCount" : 4, "modifiedCount" : 4 }
```

**Comment by wixaw** [comment]
Yes, it's work ! 
Thank's you so much

**Comment by mserranom** [comment]
> Ah yes ! i activated beta to help a collaborator !

Out of curiosity, did you activated it recently? Do you recall the actual features you enabled/disabled with the beta flags? 

Thank you!

**Comment by wixaw** [comment]
I activated with web interface it on July 10 with version 2.7.1 because I received two tickets

- 1st :
I come back to you regarding the "comments" feature of sharelatex. This allows you to leave comments on part of the latex text/code. Although this feature is visible on irit's sharelatex, it produces the following error when I submit a comment:

Error submitting comment. Sorry, there was a problem submitting your comment

= > but I could not answer because I said that I did not support a beta function

-2nd : 
we would be very interested in being able to use the functionality documented in https://www.overleaf.com/learn/how-to/Using_Git_and_GitHub (namely the functionality of Git-Bridge, which makes it possible to transform an Overleaf/Sharelatex project into a remote git …)

=> I wanted to see if it was available with beta
